Tasks/AzurePowerShellV4/TryMakingModuleAvailable.ps1

```diff
@@ -0,0 +1,78 @@
+[CmdletBinding()]
+param (
+    [string]
+    $targetVersion,
+
+    [string] [Parameter(Mandatory = $true)] [ValidateSet("Windows", "Linux", "Mac")]
+    $platform
+)
+
+try {
+    $isWin = $platform -eq "Windows"
+    if ($isWin) {
```

Comment:
Can you add a comment explaining what each of these checks is trying to achieve.

Reply:
I've added write-verbose for each of the step, that should suffice.